web/src/jobs/sessions.rs

```diff
@@ -0,0 +1,28 @@
+use cja::{app_state::AppState as _, jobs::Job};
+use miette::IntoDiagnostic;
+
+use crate::AppState;
+
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub struct Cleanup;
+
+#[async_trait::async_trait]
+impl Job<AppState> for Cleanup {
+    const NAME: &'static str = "sessions::cleanup";
+
+    async fn run(&self, app_state: AppState) -> miette::Result<()> {
+        let db = app_state.db();
+
+        sqlx::query!(
+            r#"
+            DELETE FROM sessions
+            WHERE expires_at < now() - interval '1 day'
+            "#,
+        )
+        .execute(db)
+        .await
+        .into_diagnostic()?;
```

Comment:
Inside jobs you will need to use `into_diagnostic()` before you use `?`

This is cause this uses `miette` instead of `color_eyre`